### ISSUE TITLE
Investigating a user query: Variant does not map to the genome

### ISSUE THREAD

**Issue by buniello** [EBISPOT/goci#637]
The user Dan Bolser reported a strange behavior for this variant:
https://www.ebi.ac.uk/gwas/variants/rs11385942

EMAIL TEXT BELOW:
On this page:
https://www.ebi.ac.uk/gwas/variants/rs11385942

you say that, "Variant does not map to the genome", however, in the LD plot below, it clearly has a genomic position.

Sorry if this is just an artifact.

Also, it is linked to Ensembl with a genomic location here:
http://www.ensembl.org/Homo_sapiens/Variation/Explore?r=3:45834468-45835469;v=rs11385942;vdb=variation;vf=95314084

Sorry if I'm just getting confused.

Many thanks,
Dan.

**Comment by buniello** [comment]
The user Dan Bolser asked for an update on this on Fri 20-05-22

**Comment by CholoTook** [comment]
Any news on this? The page mentioned is still broked.

**Comment by CholoTook** [comment]
Thank you for this update (and this fix!).

I clicked around a bit and ended up on this page:
https://www.ebi.ac.uk/gwas/variants/chr12:62070198

Again, this appears to be unmapped...

However, I googled and found this:
https://www.medrxiv.org/content/medrxiv/early/2021/05/14/2020.11.10.20229203/DC1/embed/media-1.pdf

Which shows rs10877786 maps to chr12:62070198:T:G.

This is consistent with the data here:
https://www.ncbi.nlm.nih.gov/snp/?term=rs10877786

Is there a plan / periodic process to map previously unassigned SNPs to their corresponding dbSNP IDs?


Many thanks,
Dan.
